src/_guides/language/effective-dart/usage.md

```diff
@@ -19,21 +19,58 @@ but *maintainers* of it sure will.
 
 These guidelines help you compose your program out of multiple files in a
 consistent, maintainable way. To keep these guidelines brief, they use "import"
-to cover `import` and `export` directives. The guidelines apply equally to both.
+to cover `import` and `export` directives. The guidelines apply equally to both. 
+
+### DON'T use library directives unless attaching doc comments or annotations
+
+[Library-level doc comments][] and [annotations][]
+must prepend a `library` declaration at the start of a file.
+There is no other reason to use the library directive. 
```

Comment:
I used "prepend" vs. "precede" because prepend implies attachment, which as far as I understand the comments and or annotation need to be _right_ above the library directive, like actually touching, no blank lines in between, i.e. attached. Precede is too vague; it implies they can be anywhere before, not necessarily anchored on.